plugins/chat/spec/requests/api/chat_channels_controller_spec.rb

```diff
@@ -421,7 +392,7 @@
       it "nullifies the field and doesn’t store an empty string" do
```

Comment:
The description for this and the name one are not right now. The way the `UpdateChannel` service is set up now with `before_context` doesn't allow us to blank out the name or description (name is much less likely to be needed, but description is legit).